localstack-core/localstack/services/apigateway/helpers.py

```diff
@@ -874,6 +874,9 @@ def connect_api_gateway_to_sqs(gateway_name, stage_name, queue_arn, path, accoun
                     "uri": "arn:aws:apigateway:%s:sqs:path/%s/%s"
                     % (sqs_region, sqs_account, queue_name),
                     "requestTemplates": {"application/json": template},
+                    "requestParameters": {
+                        "integration.request.header.Content-Type": "'application/x-www-form-urlencoded'"
+                    },
```

Comment:
We seem to have a few "testing only" helpers in this file. Should we make a plan to move them to the tests files?

Reply:
Yes please 😭 We really should, I believe once we migrate the current implementation into a `legacy` folder, we can find all imports to `legacy` and refactor all of it. This function for example makes no sense and is not even that right...